glimmer-scoped-css/src/ast-transform.ts

```diff
@@ -32,8 +33,30 @@ const scopedCSSTransform: ASTPluginBuilder<Env> = (env) => {
     name: 'glimmer-scoped-css',
 
     visitor: {
-      ElementNode(node) {
+      Template(node) {
+        let styleTag = node.body.find(
+          (n) => n.type === 'ElementNode' && n.tag === 'style'
+        );
+
+        if (styleTag) {
+          currentTemplateStyleHash = md5(
+            textContent(styleTag as ASTv1.ElementNode)
+          ).slice(0, 10);
+        }
+
+        return node;
+      },
+      ElementNode(node, walker) {
+        // FIXME how can we know `currentTemplate` is true?
```

Comment:
Status of this FIXME?

Reply:
I think you followed an old link, I’ve since removed these two `FIXME`s, this one is addressed by the addition of the `.gjs` file and the next is addressed by the `failing-test-app` script 🎉

Reply:
Ah, yes. I was looking at a single commit to avoid the noise of the boilerplate files. LGTM 👍